README.md

```diff
@@ -49,15 +49,14 @@ bpmnJsTracking.on('tracking.disabled', function(event) {
 
 ## Tracked events
 
-### Diagram events
-
 | Event Name | Structure |
 | :--- | :--- |
-| `diagram.select`| <ul><li>oldSelection</li><li>newSelection</li></ul>|
+| `popupMenu.open`| <ul><li>selection</li></ul>|
+| `popupMenu.trigger`| <ul><li>triggerType: ["click", "drag", "keyboard"]</li><li>entryLabel</li></ul>|
+
 
-### Controls Events
+### Modeling events
 
 | Event Name | Structure |
 | :--- | :--- |
-| `controls.popupMenu.open`| <ul><li>selection</li></ul>|
-| `controls.popupMenu.trigger`| <ul><li>triggerType: ["click", "drag", "keyboard"]</li><li>entryLabel</li></ul>|
+| `modeling.select`| <ul><li>oldSelection</li><li>newSelection</li></ul>|
```

Comment:
As this is a `selection` concern I'd rather rename this event to `selection.select`.

Reply:
Modeling would be anything that performs actual diagram changes, i.e. element deletion, property changes, ...

Selection works with modeling (and viewing).